docs/monitors/windows-iis.md

```diff
@@ -0,0 +1,113 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# windows-iis
+
+
+This monitor reports metrics for Windows Internet Information Services.
+It is used to drive the Windows IIS dashboard content.
+
+## Windows Performance Counters
+The underlying source for these metrics are Windows Performance Counters.
+Most of the performance counters that we query in this monitor are actually Gauges
+that represent rates per second and percentages.
+This monitor reports the instantaneous values for these Windows Performance Counters.
+This means that in between a collection interval, spikes could occur on the
+Performance Counters.  The best way to mitigate this limitation is to increase
+the reporting interval on this monitor to collect more frequently.
+
+Sample YAML configuration:
+
+```yaml
+monitors:
+ - type: windows-iis
+```
+
+
+Monitor Type: `windows-iis`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/windowsiis)
+
+**Accepts Endpoints**: No
+
+**Multiple Instances Allowed**: **No**
+
+## Configuration
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `counterRefreshInterval` | no | `int64` | (Windows Only) Number of seconds that wildcards in counter paths should be expanded and how often to refresh counters from configuration. (**default:** `60s`) |
+| `printValid` | no | `bool` | (Windows Only) Print out the configurations that match available performance counters.  This used for debugging. (**default:** `false`) |
+
+
+
+
+## Metrics
+
+The following table lists the metrics available for this monitor. Metrics that are not marked as Custom are standard metrics and are monitored by default.
+
+| Name | Type | Custom | Description |
+| ---  | ---  | ---    | ---         |
+| `process.handle_count` | gauge | X | The total number of handles currently open by this process. This number is equal to the sum of the handles currently open by each thread in this process. |
```

Comment:
Do any of these need to be whitelisted or are they all going to be custom?  Talk to @mstumpfx.